test/libsolidity/semanticTests/libraries/stub.sol

```diff
@@ -0,0 +1,13 @@
+library L {
+    function f(uint256 v) external returns (uint256) { return v*v; }
+}
+contract C {
+    function g(uint256 v) external returns (uint256) {
+        return L.f(v);
+    }
+}
+// ----
+// library: L
+// g(uint256): 1 -> 1
+// g(uint256): 2 -> 4
+// g(uint256): 4 -> 16
```

Comment:
Could it also call `f` directly?

Reply:
Nope, I don't think so.

Reply:
As in - the calls always go to the last contract in the source, which is C not L.